pynet/tests/test_pynet.py

```diff
@@ -0,0 +1,103 @@
+import unittest
+
+import tensorflow as tf
+
+from pynet.model.modules import level_0, level_1, level_2, level_3, level_4, level_5
+from pynet.model.pynet import PyNet
+
+
+class ModelTester(unittest.TestCase):
```

Comment:
Let's make this similar to LossTester class, more efficient and readable.
E.g.
```suggestion
class ModelTester(unittest.TestCase):
    def __init__(self, methodName: str = "runTest") -> None:
        super().__init__(methodName)
        l4_pool = tf.ones((1, 14, 14, 256))
        l4_out_1 = tf.ones((1,28,28,256))
        l5_pass_a = tf.ones((1,28,28,256))
        l5_pass_b = tf.ones((1,28,28,256))
        ...
```